File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        elems = {}

        for c in nums:
            elems[c] = elems.get(c, 0) + 1
        
        for e in elems:
            if elems[e] > len(nums) / 2:
                return e

File: Data Structures & Algorithms/maximum-subarray/submission-0.java
class Solution {
    public int maxSubArray(int[] nums) {
        int max = nums[0];
        int maxendingati = nums[0];

        for(int i = 1; i < nums.length; i++) {
            maxendingati = nums[i] > nums[i] + maxendingati? nums[i] : nums[i] + maxendingati;
            max = max > maxendingati? max : maxendingati;
        }

        return max;
    }
}
